docs/languages/Java.md

```diff
@@ -16,31 +17,39 @@ There are special use-cases that each language supports; this document pertains
 <!-- tocstop -->
 
 ## Include equals function for the class
+
 To overwrite the `equal` method, use the preset `JAVA_COMMON_PRESET` and provide the option `equal: true`.
 
 Check out this [example for a live demonstration](../../examples/java-generate-equals).
 
-
 ## Include hashCode function for the class
+
 TODO
 
 ## Change the collection type for arrays
-TODO
+
+Sometimes, we might want to render the collections as `List` type. To do so, prove the option `collectionType: 'List'`.
```

Comment:
Maybe something along the lines of this is better 🤔 :
```suggestion
Sometimes, we might want to render a different collection type, and instead of the default `Array` use `List` type. To do so, provide the option `collectionType: 'List'`.
```